client/html/member.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<body>
+    {{template "index" .}}
+    <main>
+        {{.picture}}
+        <h1>{{.name}}</h1>
+        <h2>{{.title}}</h2>
+        <p>{{.status}}</p>
+
+        <h3>Artiklar:</h3>
+        {{range .articles}}
+            <a href={{.URL}}>
```

Comment:
```suggestion
            <a href="{{.URL}}">
```
pretty sure?? or maybe the go templating thing just fixes it